docs/sdk_reference.fbmd

```diff
@@ -172,6 +172,10 @@ Graph nodes are collections that represent nodes returned by the Graph API. And
       '[`Facebook\\GraphNodes\\GraphUser`](/docs/php/GraphNode#user-instance-methods)',
       'A collection that represents a User node.',
     ],
+    [
+      '[`Facebook\\GraphNodes\\Birthday`](/docs/php/Birthday#instance-methods)',
```

Comment:
I agree. I think I'd modify the docs on `GraphNode` and edit the lines that [talk about the return type](https://github.com/facebook/facebook-php-sdk-v4/blob/master/docs/GraphNode.fbmd#L167-L171) and link to the `Birthday` documentation from there. :)